common/web/types/src/util/compiler-interfaces.ts

```diff
@@ -32,6 +32,18 @@ export function compilerErrorSeverityName(code: number): string {
   }
 }
 
+/**
+ * Format the error code number
+ * example: "FATAL:0x03004"
+ */
+export function compilerErrorFormatCode(code: number): string {
+  const severity = code & CompilerErrorSeverity.Severity_Mask;
+  const severityName = compilerErrorSeverityName(severity);
+  const errorCode = code & CompilerErrorSeverity.Error_Mask;
+  const errorCodeString = Number(errorCode).toString(16).padStart(5,'0');
+  return `${severityName}:0x${errorCodeString}`;
+}
```

Comment:
This is fine for now but may need updating as we consolidate the error reporting further as part of #5283. It doesn't match what is generated from NodeCompilerCallbacks at this point (which itself needs work):

https://github.com/keymanapp/keyman/blob/7746cadd980acba251788ab3800ed37874563c9f/developer/src/kmc/src/messages/NodeCompilerCallbacks.ts#L31-L38

Reply:
Didn't see that. I looked in common and didn't see such a function. Yes it would make sense to consolidate. 

The use case was trying to track down why the wrong err was thrown.